### PR TITLE
added app_ prefix to form type names

### PR DESCRIPTION
**Pull request by OskarStark**
refs #5077

I'm not really sure about the changes of the `alias` names... 

ping @xabbuh

Greetings from Symfony Live Berlin 2015 - Hack Day :smile:

### CODE REVIEW

File: book/forms.rst [hunk 1]
```diff
@@ -1173,7 +1173,7 @@ easy to use in your application.
 
         # src/AppBundle/Resources/config/services.yml
         services:
-            app.form.type.task:
+            app.form.type.app_task:
```
Comment (xabbuh) [inline book/forms.rst]: I would not change this here (we already have the `app.` prefix).

File: book/forms.rst [hunk 2]
```diff
@@ -1173,7 +1173,7 @@ easy to use in your application.
...
                 class: AppBundle\Form\Type\TaskType
                 tags:
                     - { name: form.type, alias: task }
```
Comment (xabbuh) [inline book/forms.rst]: missing `app_` prefix here

File: book/forms.rst (hunk 1 above)
Comment (xabbuh) [inline book/forms.rst]: same below

**Comment by xabbuh** [comment]
We should make this change in the `2.3` branch (the `getName()` method and the `alias` configuration will be gone in Symfony 3.0).

File: book/forms.rst (hunk 2 above)
Comment (OskarStark) [inline book/forms.rst]: done

File: book/forms.rst (hunk 1 above)
Comment (OskarStark) [inline book/forms.rst]: ok, i changed the code

**Comment by OskarStark** [comment]
any news on this PR?

File: cookbook/form/create_custom_field_type.rst [hunk 3]
```diff
@@ -311,14 +311,14 @@ the ``genders`` parameter value as the first argument to its to-be-created
                 arguments:
                     - "%genders%"
                 tags:
-                    - { name: form.type, alias: gender }
+                    - { name: form.type, alias: app_gender }
```
Comment (xabbuh) [inline cookbook/form/create_custom_field_type.rst]: The `getName()` bundle at the top of the article must be updated too.

**Comment by WouterJ** [comment]
I've rebased this PR on top of 2.3 and updated the `getName()` method as indicated by @xabbuh in https://github.com/symfony/symfony-docs/pull/6049

Thank you for fixing this @OskarStark! :christmas_tree:

**Comment by OskarStark** [comment]
Good Job :+1:

Am Samstag, 19. Dezember 2015 schrieb Wouter J :

**Pull request closed without merging**